blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -150,6 +150,7 @@
     
     // how long we typically wait to schedule a rollup.
     ROLLUP_DELAY_MILLIS("300000"),
+    DELAYED_METRICS_ROLLUP_DELAY_MILLIS("300000"),
```

Comment:
But why do we want to set the default for DELAYED_METRICS_ROLLUP_DELAY_MILLIS be the same as normal?